chapters/sampling-randomization-power.tex

```diff
@@ -1,266 +1,611 @@
+
+You might think that you are able to keep all of these details in your head,
+but your whole research team is unlikely to have the same understanding,
+at all times, of all the datasets required.
+The only way to make sure that the full team shares the same understanding
+is to write it down in what is called a \textbf{data map}.\sidenote{
+	\url{https://dimewiki.worldbank.org/data\_map} (TO BE CREATED)}
+\index{data maps}
+
+%% THIS IS A WEIRD 2 PARAGRAPHS WITH JUST A LIST OF QUESTIONS
+A data map is more than just a list of datasets.
+Its purpose is to specify the characteristics and linkages of those datasets.
+Some basic characteristics should filled in during the planning stage.
+Example of those characteristics are:
+What is the unit of observation for each dataset?
+What is the name of the ID variable that uniquely and fully identifies each dataset?
+Should any datasets be possible to merge one-to-one?
+And if so, will the have the same unique identifier?
+Should any datasets be possible to merge many-to-one (for example school and student)?
+And if so, what identifier will be used?
+Solving all of these questions before you acquire any data,
+and making sure that the full research team knows where to find this info,
+will save you a ton of time later on.
+
+Other characteristics should be updated as the project progresses.
+Example of those characteristics are:
+Where will the data be obtain from?
+Where in the project folder are the raw original data stored
+and where are back-ups of the raw data stored?
+See the wiki article linked to above for an template that you can use for your project.
+
+If you have multiple datasets with the same unit of observation
+then you should create a \textbf{master dataset}\sidenote{
+	\url{https://dimewiki.worldbank.org/Master\_Data\_Set}}
+for each of those units of observation and include that in the required datasets.
+The master dataset should include an ID variable
+that can be used to link or merge all datasets with that unit of observation.
+It should also list project wide time-invariant information
+like sampling and treatment status.
+Additionally, any unit of observation
+that will be used in sampling or treatment assignment
+should have a master dataset,
+and that dataset should be used when sampling or assigning treatment.
+
+\section{Taking research design into account when planing your data work}
+
+The most important steps in data work planning
+to correctly translate a research design into data work
+are sampling and treatment assignment.
+Therefore, understanding and correctly implementing sampling and treatment assignment
+is essential to ensuring that data is acquired properly and that research
+is correctly carried out in the field, so that the results
+are valid interpretations of your research question.
+
+Not all projects need to do sampling and treatment assignment.
+Sampling is almost always required when you collect your own data,
```

Comment:
following workflow suggestion above, i think makes sense to organize content so we talk about treatment assignment first and then sampling

Reply:
I think sampling should come before for the following reasons:
* Sampling is more general. Both as a concept in science in general, but also that it is applicable to more projects of the audience of this book. Sampling is almost always needed, but random assignment is only experimental design.
* I am not sure if assignment always comes before sampling. Often we for example, sample 100 schools, and then we randomize which teachers will get the treatment. Also, we could collect baseline data in order to have strata and other info to do advanced random assignment on.

Since there is variation in which comes first, then I think sampling should come first as it is more general.

Reply:
sounds fine to me

Reply:
this is resolved then, right?